README.md

```diff
@@ -71,6 +71,45 @@ Then the following data is inserted in the store:
         as:deleted "thetimeofdelete"^^xsd:dateTime  .
 ```
 
+## Extending Forms
+
+### Predicates and Types
+
+Use the same model as regular forms, with the following differences:
+-   don't use the `sh:group` predicate to denote connections to the owning form (or its sections/listings). Instead use `ext:extendsGroup`
+-   don't use the `form:includes` predicate to denote that a field is part of a form. Instead use `ext:extendsForm`
+
+An extension is represented by 
+
+    <extensionUri> a form:Extension ;
+                   form:includes <fieldUri> ;
+                   form:extends <formUri> ;
+                   mu:uuid "b5a86f3a-aac8-4911-a3fb-37f9f194b58e" .
+
+### Order
+
+We want fields to be added to the original form in a specific order, intermixed with the existing fields (before, after, between). Unfortunately, the order property is an integer and loses float/double precision if it is added to it. Therefore we should leave gaps of e.g. 100 between the order properties of our forms.
+
+### Extending the form with direct properties
+
+Some properties are defined directly on the form, e.g. the generator. If we want to extend the form with such properties, we should define them on the `form:Extension`. The form-content service will take the `form:Extension` instance and treat it as a `form:Form` instance. It will examine the form instance it extends and add all of its predicates to itself. 
```

Comment:
The text is clear, but I have a bit of difficulty actually visualising this, maybe it is a good idea to add some example forms to this service in the future, like in the ember-submission-frorm-fields repo?

Reply:
yes! there is a story for this in this sprint!  please pick it up! it would be good that someone other than me writes the example so the spec is proven to be clear enough